extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -92,6 +94,16 @@ public void handleMessage(Object msg) {
                     HttpResponse res = (HttpResponse) msg;
                     responseBuilder.setStatusCode(res.status().code());
 
+                    final HttpHeaders responseHeaders = res.headers();
+                    if (responseHeaders != null) {
+                        final String contentType = responseHeaders.getAsString(HttpHeaderNames.CONTENT_TYPE);
+                        if (contentType != null && contentType.trim().length() > 0) {
+                            final Map<String, String> headers = new HashMap<>();
+                            headers.put("Content-Type", contentType);
```

Comment:
Shouldn't you use the constant here too?

Also could you squash the commits? Thanks!

Reply:
I wasn't sure about using the constant there too. The constant is an `AsciiString`, not a `String` so I need to call `toString()` on it. In addition, the constant is from Netty, whereas the `Map` is being put into AWS. Very unlikely I know, but if the Netty headers started being named differently, in a Netty-specific way, it might not work properly on AWS.

If you still want me to use the constant, I can:

`headers.put(HttpHeaderNames.CONTENT_TYPE.toString(), contentType);`

Once you've confirmed one way or the other, I'll work out how to squash the commits!

Reply:
Yeah, keep it as it is, I totally agree with you it's better this way.